soh/soh/Enhancements/randomizer/randomizerTypes.h

```diff
@@ -1356,6 +1357,12 @@ typedef enum {
     RO_LOGIC_MAX,
 } RandoOptionLogic;
 
+typedef enum {
+    RO_ALL_LOCATIONS_REACHABLE_OFF,
+    RO_ALL_LOCATIONS_REACHABLE_ON,
+    RO_ALL_LOCATIONS_REACHABLE_MAX,
+} RandoOptionAllLocationsReachable;
```

Comment:
Do we need this? Can probably just use `RandoOptionGenericOffOn`

Reply:
you right, done 👍